src/Middleware/HttpsPolicy/src/HttpsRedirectionMiddleware.cs

```diff
@@ -165,5 +166,8 @@ private int TryGetHttpsPort()
 
         _logger.FailedToDeterminePort();
         return PortNotFound;
+
+        [UnconditionalSuppressMessage("Trimming", "IL2026", Justification = "Primitive type")]
+        int? GetIntConfigValue(string key) => _config.GetValue<int?>(key);
```

Comment:
I know this might work but we should just do this manually, ala https://github.com/microsoft/reverse-proxy/blob/7d8b8152b1c67060f307066219515d9985164329/src/ReverseProxy/Configuration/ConfigProvider/ConfigurationReadingExtensions.cs#L14-L17